change/@microsoft-teams-js-564f452b-e760-4a7a-bc80-fbc7288b70fb.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "major",
+  "comment": "Git magic week of 4/18/2022 \n1. IMeetingDetails has been renamed to IMeetingDetailsResponse and IDetails has been replaced with IMeetingOrCallDetailsBase<T>. As such, meeting.getMeetingDetails() now returns Promise<IMeetingDetailsResponse> rather than Promise<IMeetingDetails>. The deprecated version also takes in a callback which takes in IMeetingDetailsResponse rather than IMeetingDetails. \n2. pages.returnFocus() now works in any FrameContext rather than just with content.",
```

Comment:
After you pull 2.0-preview, this comment will have to change since we don't return promises from APIs in meeting.ts anymore.

Reply:
Another great catch, fixed in latest commits!